src/docs/product/alerts/notifications/notification-settings.mdx

```diff
@@ -1,88 +1,28 @@
 ---
 title: Personal Notification Settings
 sidebar_order:
-description: "Learn how to manage the notifications that Sentry sends you."
+description: "Learn how to manage your personal notifications settings."
 ---
 
-You can adjust your personal [workflow](#workflow) and [deploy](#deploy) notifications, as well as your personal issue alert settings, in your [account settings](https://sentry.io/settings/account/notifications/). Manage your notifications by navigating to **User Settings > Notifications**. You **cannot configure** quota notifications.
+You can update your personal notification settings for things like [workflow](/product/alerts/notifications/), deploy, issue alert, and [spike protection](/product/accounts/quotas/spike-protection/) by turning them on or off, specifying how you'd like to be notified (via email, Slack, or both), and more. Users who **aren't** on an [Enterprise](https://sentry.io/pricing/) plan can also configure their quota notifications.
 
-## Alerts
+![Issue alert notification page](issue-alert-notification.png)
 
-<Note>
+To make updates, go to your [user settings](https://sentry.io/settings/account/notifications/), select "Notifications", then click on the notification type you want to update.
 
-This setting does not affect alerts that are configured to send to your email explicitly.
+![Issue alert notification detail page](issue-alert-notification-detail.png)
 
-</Note>
+## Notification Types
 
-In **Notifications**, you can toggle [issue alert](/product/alerts/alert-types/#issue-alerts) notifications on and off, globally.
+Here's a list of all the personal notification types you can update:
 
-You can also fine tune your alert notifications on a per-project basis by choosing "Default”, “On”, or “Off”. If you select “Default”, the setting for the project will be the same as your global setting.
-
-### Delivery Method
-
-You can decide where you receive personal alert notifications by choosing from the options:
-
-- Send to Email
-- Send to [Slack](/product/integrations/notification-incidents/slack/)
-- Send to Email and Slack
-
-[Slack](/product/integrations/notification-incidents/slack/) is only available as a delivery method if your organization has the integration installed and your [Slack identity is linked to your Sentry account](/product/integrations/notification-incidents/slack/#linking-your-slack-and-sentry-accounts).
-
-## Workflow
-
-The global settings for workflow notifications are:
-
-- Always
-- Only On Issues I Subscribe To
```

Comment:
@lizokm why did we remove this bit?

Reply:
This functionality is clear if you go to the product and I don't think we add anything in docs by explaining something that's already easy to understand in the product itself. Open to differing opinions :)

Reply:
@lizokm I think a user might be confused about what it means to be subscribed to an issue, I'm not sure how intuitive that is

Reply:
Agreed that subscriptions are confusing, but we don't really explain what it means to be subscribed in this section?

Reply:
Ah, I see what you mean. We can just add this section from the main notification page: You receive workflow notifications when you’re subscribed to an issue, and you subscribe to issues by:

Clicking the subscribe bell icon on an issue
Involvement in a commit related to the issue
Commenting on or bookmarking the issue
Mention of you or your team in the issue
You or your team being assigned to the issue
Archiving an issue until it escalates